k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -534,7 +539,14 @@ private boolean initFromIntent(final Intent intent) {
             }
 
             recipientPresenter.initFromSendOrViewIntent(intent);
+        }
 
+        if ((BuildConfig.APPLICATION_ID + ".AUTOCRYPT_PEER_ACTION").equals(action)) {
```

Comment:
Needs to be changed to "org.autocrypt.PEER_ACTION".

Reply:
oh, indeed. fixing that